### PR TITLE
feat: Add JSON Cleaner Component

### PR DESCRIPTION
**Pull request by h-arnold**
Many of my flows involve the manipulation of JSON objects generated or modified by LLMs. Despite my best prompting efforts, sometimes LLMs still preface their JSON objects with: ```'Sure I can create that JSON object for you...  ``` and breaking the code downstream that needs to process that data. Even if they only provide the JSON object, sometimes they make tiny errors or use characters not allowed as part of the JSON spec.  

To fix this, I created a custom component that makes use of the  [json_repair library](https://github.com/mangiucugna/json_repair) to clean up messy JSON strings and ensure that the output is always compliant, regardless of which LLM you use. 

It has a couple of simple cleansing steps beforehand as ```json_repair``` doesn't always trim the JSON strings if there's too much text before or after the JSON string. I've found that outputs from Google Gemini and Mistral sometimes output invalid control characters, which is the 2nd step.

A PR request rather than a store submission is necessary because the backend needs the ```json_repair``` library installed to work.

### CODE REVIEW

File: src/backend/base/langflow/components/prototypes/JSONCleaner.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import json
+import json_repair #type: ignore
```
Comment (nicoloboschi) [inline src/backend/base/langflow/components/prototypes/JSONCleaner.py:2]: can you put it in the `clean_json` method? with safe try/except

similar to https://github.com/langflow-ai/langflow/blob/74433bf023a66a0e7b98cb8eec0f14a1f39e5404/src/backend/base/langflow/components/vectorstores/AstraDB.py#L170-L177

Comment (h-arnold) [reply src/backend/base/langflow/components/prototypes/JSONCleaner.py]: All done! :)

**Review by ogabrielluiz — APPROVED** [review]
Hey @h-arnold 

I like this one. I might even use json-repair in the webhook component.

LGTM

**Comment by h-arnold** [comment]
Hey @ogabrielluiz!

Glad you like it. :) Thanks for all your hard work with langflow!

**Comment by h-arnold** [comment]
@ogabrielluiz 

Apologies for the notification spam. I tried sorting out the merge from my phone which ended up breaking things more!

The PR should be good to merge now. :)

**Pull request merged**